content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -394,3 +394,15 @@ For more details on etcd upgrade, please refer to the [etcd upgrades](https://et
 {{< note >}}
 Before you start an upgrade, please back up your etcd cluster first.
 {{< /note >}}
+
+## Maintaining etcd clusters
+
+Fore more details on etcd maintenance, please refer to the [etcd maintenance](https://etcd.io/docs/latest/op-guide/maintenance/) documentation.
+
+{{< note >}}
+Defragmentation is an expensive operation, so it should be executed as infrequent
+as possible. On the other hand, it's also necessary to make sure any etcd member
+will not run out of the storage quota. It's recommended to run defragmentation
+using [etcd-defrag](https://github.com/ahrtr/etcd-defrag), which is an easier to
```

Comment:
i think the sig docs rules had something about a third party shortcode / disclaimer that has to be added here, but i don't remember the details.

Reply:
In order of preference:
1. do not make a specific recommendation [following [https://kubernetes.io/blog/2020/05/third-party-dual-sourced-content/](https://kubernetes.io/blog/2020/05/third-party-dual-sourced-content/)] - etcd is not part of the K8s project
2. link to external docs that recommend the tool.
3. add a third party content disclaimer shortcode

Please pick one of these.

Until then:
/lgtm cancel

(fine to LGTM it back once that's addressed)

Reply:
> 3\. add a third party content disclaimer shortcode

the shortcode was:
`{{% thirdparty-content %}}`

Reply:
Thanks both & updated.

Reply:
@neolit123 @sftim PTAL, thx